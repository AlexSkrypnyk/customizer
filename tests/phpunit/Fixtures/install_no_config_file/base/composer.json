{
    "name": "yourorg/yourtempaltepackage",
    "description": "Your template package description",
    "type": "project",
    "license": "proprietary",
    "require": {
        "php": ">=8.2",
        "monolog/monolog": "^2.0"
    },
    "require-dev": {
        "alexskrypnyk/customizer": "*"
    },
    "config": {
        "allow-plugins": {
            "alexskrypnyk/customizer": true
        }
    }
}
