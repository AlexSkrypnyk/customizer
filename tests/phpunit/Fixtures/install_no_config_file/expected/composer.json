{
    "name": "yourorg/yourtempaltepackage",
    "description": "Your template package description",
    "type": "project",
    "license": "proprietary",
    "require": {
        "php": ">=8.2",
        "monolog/monolog": "^2.0"
    }
}
